AirCasting/Dashboard/DashboardView.swift

```diff
@@ -14,15 +14,16 @@ import Resolver
 struct DashboardView: View {
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
-    @EnvironmentObject var selectedSection: SelectSection
+    // TODO: - We can remove it in a future
+    @EnvironmentObject var selectedSection_: SelectSection
     @EnvironmentObject var reorderButton: ReorderButton
     @EnvironmentObject var searchAndFollowButton: SearchAndFollowButton
     @State var isRefreshing: Bool = false
+    @State var selectedSection: SelectedSection = .following
     @Injected private var sessionSynchronizer: SessionSynchronizer
+    @Injected private var persistenceController: PersistenceController
     @InjectedObject private var featureFlagsViewModel: FeatureFlagsViewModel
```

Comment:
Is that still needed?

Reply:
I think it is needed because we use it for init SessionListView.

![Screenshot 2022-06-10 at 07 41 41](https://user-images.githubusercontent.com/82096200/172998170-899bce30-8b97-43bc-bfcf-baab156b5cd6.png)